packages/terra-disclosure-manager/docs/README.md

```diff
@@ -154,18 +154,18 @@ disclosureManager.disclose({
 
 `disclose` Argument API:
 
-|Key|Value|
-|---|---|
-|`preferredType`|The String describing the preferred disclosure type. This will be used to match the disclosure request to an appropriate DisclosureManager. Depending on the structuring of managers in a given component, the `preferredType` value may not be honored. If the provided `preferredType` is not supported by any present disclosure managers, the root disclosure manager will perform the disclosure using its own disclosure type, regardless of the `preferredType` value.
-|`size`|The String size desired for the disclosure. One of `tiny`, `small`, `medium`, `large`, `huge`, or `fullscreen`. The functional implementation of this size is determined by the rendering component.|
-|`content`|An Object containing a key and a component describing the component to be disclosed. See the `content` API below.|
+|Key|Is Required|Value|
+|---|---|---|
+|`preferredType`|optional|The String describing the preferred disclosure type. This will be used to match the disclosure request to an appropriate DisclosureManager. Depending on the structuring of managers in a given component, the `preferredType` value may not be honored. If the provided `preferredType` is not supported by any present disclosure managers, the root disclosure manager will perform the disclosure using its own disclosure type, regardless of the `preferredType` value.|
+|`size`|optional|The String size desired for the disclosure. One of `tiny`, `small`, `medium`, `large`, `huge`, or `fullscreen`. The functional implementation of this size is determined by the rendering component. If not provided, a default size will be used.|
```

Comment:
I think `dimensions` should also be documented. 

https://github.com/cerner/terra-framework/blob/ac87285ac89dbc005e5f3483c0cc863585bd0191/packages/terra-disclosure-manager/src/DisclosureManager.jsx#L245

Reply:
you got it 1b87cb7